src/Components/CreateImageWizard/ValidatedTextInput.tsx

```diff
@@ -30,6 +34,65 @@ interface HookValidatedTextInputPropTypes extends TextInputProps {
   warning?: string;
 }
 
+interface HookPasswordValidatedTextInputPropTypes
+  extends HookValidatedTextInputPropTypes {
+  isPasswordVisible?: boolean;
+  togglePasswordVisibility?: () => void;
+}
+
+export const HookPasswordValidatedInput = ({
+  ariaLabel,
+  placeholder,
+  dataTestId,
+  value,
+  ouiaId,
+  stepValidation,
+  fieldName,
+  onChange,
+  type = 'text',
+  isPasswordVisible,
+  togglePasswordVisibility,
```

Comment:
See my comment below - I think we can remove the `isPasswordVisible` and `togglePasswordVisibility` props and just keep track of their state inside of this component.